packages/typespec-ts/README.md

```diff
@@ -145,6 +145,14 @@ addCredentials: true
 credentialKeyHeaderName: Your-Subscription-Key
 ```
 
+### clearOutputFolder
+
+If we enable this option `clearOutputFolder` we would empty the whole output folder. By default we only empty the sources folder which means any metadata files will not be removed if it is at project root. This would be useful in pipeline.
+
+```yaml
+clearOutputFolder: true
```

Comment:
@wanlwanl Qiaoqiao and I discussed that we could always enable `clearOutputFolder: true` to empty project folder to avoid in-compitible customization code issue.

/cc @qiaozha

Reply:
If automation side could delete the output folder by default, then I guess this option can be omit in the codegen automation config? But I agree this option is necessary for codegen.

Reply:
FYI, automation side always clear output folder by default for now.

Reply:
thanks for this info and i agreed with @qiaozha as codegen we should have this option.